cdk/lib/riffraff.test.ts

```diff
@@ -0,0 +1,10 @@
+import { riffraff } from '../bin/cdk';
+
+describe('generated riff-raff.yaml', () => {
+	it('matches the snapshot', () => {
+		// @ts-ignore
+		const outdir = riffraff.outdir; // this changes for every test execution and best not to change cdk.ts too much
```

Comment:
minor: this property is set from the `outdir` of the parent `App`. If we were to instantiate an `App` in the test, we could set the `outdir` prop and thus become deterministic. For example https://github.com/guardian/cdk/blob/ec0fb851e52d6c2b966b6b264eb21dc6a6186cfe/src/riff-raff-yaml-file/index.test.ts#L16.

Reply:
I prefer the idea of snapshot testing the actual stacks we create, rather than some arbitrarily configured test one.